tests/common/helpers/sonic_db.py

```diff
@@ -545,15 +545,19 @@ class SonicDbNoCommandOutput(Exception):
     pass
 
 
-def redis_get_keys(duthost, db_id, pattern):
+def redis_get_keys(duthost, db_id, pattern, asic_id=None):
     """
     Get all keys for a given pattern in given redis database
     :param duthost: DUT host object
     :param db_id: ID of redis database
     :param pattern: Redis key pattern
     :return: A list of key name in string
     """
-    cmd = 'sonic-db-cli {} KEYS \"{}\"'.format(db_id, pattern)
+    if asic_id is not None:
```

Comment:
can we use the function in `sonic_asic` iso add if asic_id checks here. The sonic_asic takes care of this.

https://github.com/sonic-net/sonic-mgmt/blob/f26c1c8f4b96f9d011ed2fe2ac78921062d79164/tests/common/devices/sonic_asic.py#L46C59-L46C72